### ISSUE TITLE
Select2Option.Classes ignored

### ISSUE THREAD

**Issue by cniemann88** [plantain-00/select2-component#34]
#### Version(if relevant): 1.0.0
5.8.4
#### Environment(if relevant):
Windows, React.
#### Code(if relevant):

```
let option: Select2Option = {
                value: 1),
                label: 'special option with class',
                classes: 'special-class-to-apply'
            }
```

#### Expected:
That the 'classes' string will be applied to the option.
#### Actual:
'Classes' doesnt seem to be taken into account when rendering the options.

**Comment by plantain-00** [comment]
`classes` is not implemented in v5.8.
It should be available in v5.9.0

**Comment by cniemann88** [comment]
works in 5.9.0
